### ISSUE TITLE
Variable false positive

### ISSUE THREAD

**Issue by jeff-hykin** [microsoft/pylance-release#520]
You can close this for not filling out the template (I'm personally not bothered by the issue) just thought maintainers might want to know

False positive shown in editor, code correctness/execution shown in REPL/terminal
```
flatten = lambda *n: (e for a in n for e in (flatten(*a) if isinstance(a, (tuple, list)) else (a,)))
```

![Screen Shot 2020-10-25 at 6 09 34 PM](https://user-images.githubusercontent.com/17692058/97121634-d619b000-16ed-11eb-9321-2199798469c4.png)

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release of pylance.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2020.10.3, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/master/CHANGELOG.md#2020103-28-october-2020
